project1.html

```diff
@@ -110,11 +110,35 @@
     <a class="Vector2"><img src="Vector2.png" alt=""></a>
     <a class="Vector3"><img src="Vector2.png" alt=""></a>
     <div><h4 class="Frameworks">Frameworks</h4></div>
-  <a class="Separator1"><img src="Separator Bottom.jpg" alt=""></a></div>
+  <a class="Separator1"><img src="SeparatorBottom.jpg" alt=""></a></div>
   <div><h4 class="Skills">Skills</h4></div>
-    <a class="Separator2"><img src="Separator Bottom.jpg" alt=""></a>
-    <a class="Separator1"><img src="Separator Bottom.jpg" alt=""></a>
+    <a class="Separator2"><img src="SeparatorBottom.jpg" alt=""></a>
+    <a class="Separator1"><img src="SeparatorBottom.jpg" alt=""></a>
 </div>    
+
+<div class="add-contact add-contact-bckgrnd">
```

Comment:
- [ ] Kindly ensure your design matches the design given, below is a comparison so that you can easily identify the differences:

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/177401356-9e6c731b-cfa8-40ed-ba14-1b9a7e710f5c.png) |  ![image](https://user-images.githubusercontent.com/61408860/177401384-a2e917ce-e187-4e5c-8d14-c6134dc01f3b.png)

- [ ] [optional] I noticed that your page is aligned on the right side of the page with a fixed width, I would suggest you change it so that it will be easier to implement the desktop version, also it will make your design neat align it to the left and give it a mobile width.